esp/services/ws_dfu/ws_dfuService.cpp

```diff
@@ -2297,8 +2297,7 @@ void CWsDfuEx::doGetFileDetails(IEspContext &context, IUserDescriptor *udesc, co
                 FileDetails.setCompressedFileSize(compressedSize);
                 if (version >= 1.34)
                 {
-                    __int64 diff = size - compressedSize;
-                    Decimal d(((double) diff)/size*100);
+                    Decimal d(((double) compressedSize)/size*100);
```

Comment:
This means that if orignal is 100 and compressed is 25 , it would currently display "Percent Compressed: 25" , which will be confusing I think.

Perhaps should change to "Compression ratio: ", calculated as uncompressed/compressed, meaning above 1 is good (acheived compression), below 1 means cause inflation.

Reply:
Not sure how confusing it will be. Reporting a ratio means to help users. Both Filesize and CompressedFileSize are also reported.

Reply:
FYI ECL Watch is dropping the Pct and showing the fileSize and fileCompressedSize side by side - we may add a little visual (calculated on the client showing the diff)